README.md

```diff
@@ -58,6 +58,44 @@ default unless you set the `QT_API` environment variable.
 * `pyside6` (to use PySide6).
 
 
+### Methods, helpers and QtPy namespace specifics
+
+As part of providing compatibility for the different Python Qt bindings available, QtPy does some modules mapping and exposes a `compat` module. Bellow you can check some elements to take into account when using QtPy.
+
+#### Module aliases and constants
```

Comment:
```suggestion
### Module aliases and constants
```

Instead of nesting these important sections four levels deep and including a long block of summary text that doesn't really add much important information or serve the purpose of being [Reference docs](https://diataxis.fr/reference/), I suggest simply making these top-level sections.